src/helm/benchmark/run.py

```diff
@@ -276,17 +283,18 @@ def main():
         run_specs=run_specs,
         auth=auth,
         url=args.server_url,
-        local=args.local,
+        local=bool(args.local),
```

Comment:
I assume this is to deal with these values being `None`...could we set `default=False` on the args to avoid the cast?  `bool(...)` seems a bit blunt...